### PR TITLE
RATEST-219: Removing an unneeded code line from addPatientAppointment() method

### PR DESCRIPTION
**Pull request by kdaud**
Ticket [ID](https://issues.openmrs.org/browse/RATEST-219)

Description -> Refactor the logic of addPatientAppointment test

### CODE REVIEW

**Comment by kdaud** [comment]
AddPatientAppointment test is currently randomly breaking in master build. At the moment AddPatientAppointment test and BookRequestAppointment test should pass master build during working days and am yet fixing the issue of the two tests to  pass on Sundays and Saturdays

**Comment by kdaud** [comment]
@sherrif10 could you take a look at this PR ?

File: ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java [hunk 1]
```diff
@@ -48,7 +47,6 @@ public void addPatientAppointmentTest() throws Exception {
         manageProviderSchedulesPage.enterMaximumTimeValue("09", "30");
         manageProviderSchedulesPage.clickOnEndTimeButton();
         manageProviderSchedulesPage.enterService(SERVICE_NAME);
-        manageProviderSchedulesPage.clickOnEndTimeButton();
```
Comment (sherrif10) [inline ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java:51]: Why is this line removed.

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java:51]: Its 

> Why is this line removed.

Its a bug!

After simulating the test locally, I noticed that manageProviderSchedulesPage.clickOnEndTimeButton(); should come before  manageProviderSchedulesPage.enterService(SERVICE_NAME); based on the page design. And the method is already called  [here](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/f1e0047531a97d08601ebff401050e7eb4df85fa/ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java#L49) and calling it [again ](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/f1e0047531a97d08601ebff401050e7eb4df85fa/ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java#L51) is making the test to break in master build as seen in the logs shared above!

**Review by sherrif10 — APPROVED** [review]
Thanks @kdaud   LGTM cc @dkayiwa

**Comment by dkayiwa** [comment]
Is this refactoring or removing an unneeded line?

**Comment by kdaud** [comment]
> Is this refactoring or removing an unneeded line?

Removing an unneeded code line is the right description to the commit.
